features/step_definitions/gallery_steps.rb

```diff
@@ -0,0 +1,11 @@
+Then 'I should see the photo lightbox' do
+  step %{I should see a "#blueimp-gallery" within "body"}
+end
+
+Then 'I should not see the photo lightbox' do
+  step %{I should not see a "#blueimp-gallery" within "body"}
+end
+
+Then 'I press the close lightbox link' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.